### PR TITLE
ref: code blocks around `remote modify` `sse`, etc.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
complement to #3498

### CODE REVIEW

File: content/docs/command-reference/remote/modify.md [hunk 1]
```diff
@@ -246,8 +246,8 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
-  default, no encryption is used.
+- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`).
+  By default, no encryption is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]:
```suggestion
- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`,
  `SSE-C`). By default, no encryption is used.
```

File: content/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -246,8 +246,8 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
-  default, no encryption is used.
+- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`,
+  `SSE-C`). By default, no encryption is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]: Not sure that's the right value. Asked in https://github.com/iterative/dvc.org/pull/3498#pullrequestreview-966849021 ⌛

Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]:
```suggestion
- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`).
  By default, no encryption is used.
```

File: content/docs/command-reference/remote/modify.md [hunk 3]
```diff
@@ -246,17 +246,18 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
+- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`). By
   default, no encryption is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]: What other options are there?

Does `AES256` require any other params to be set?

@dberenbaum @efiop thanks

Comment (dberenbaum) [reply content/docs/command-reference/remote/modify.md:250]: > What other options are there?

Those are the only valid values (see https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html).

> Does `AES256` require any other params to be set?

No, AWS automatically manages the keys in that case.

Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]:
```suggestion
- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms`. By
  default, no encryption is used.
```

File: content/docs/command-reference/remote/modify.md [hunk 4]
```diff
@@ -246,17 +246,18 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
+- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms`. By
   default, no encryption is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:250]:
```suggestion
- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms`
  (the latter requires setting `sse_kms_key_id` as well). By default, no
  encryption is used.
```

File: content/docs/command-reference/remote/modify.md [hunk 5]
```diff
@@ -246,17 +246,19 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
-  default, no encryption is used.
+- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms` (the
+  latter requires setting `sse_kms_key_id` as well). By default, no encryption
```
Comment (dberenbaum) [inline content/docs/command-reference/remote/modify.md:250]: Hm, I guess the docs have long said that `sse_kms_key_id` is required when using SSE-KMS, but I don't think that's true. AWS will automatically manage the keys for you if you don't provide one. See https://aws.amazon.com/premiumsupport/knowledge-center/s3-object-encryption-keys/.

File: content/docs/command-reference/remote/modify.md [hunk 6]
```diff
@@ -246,17 +246,19 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
...
+  is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:251]:
```suggestion
- `sse` - server-side encryption algorithm to use: `AES256` or `aws:kms`.
  By default, no encryption is used.
```

File: content/docs/command-reference/remote/modify.md [hunk 7]
```diff
@@ -246,17 +246,19 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
...
   ```
 
 - `sse_kms_key_id` - identifier of the key to encrypt data uploaded when using
-  SSE-KMS. Required when the `sse` parameter (above) is set to `aws:kms`. This
-  parameter will be passed directly to AWS S3 functions, so DVC supports any
-  value that S3 supports, including both key ids and aliases.
+  [SSE-KMS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html).
+  Required when `sse` is set to `aws:kms`. This parameter will be passed
+  directly to AWS S3, so DVC supports any value that S3 supports, including both
+  key IDs and aliases.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:261]:
```suggestion
  [SSE-KMS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html)
  (see `sse`). This parameter will be passed
  directly to AWS S3, so DVC supports any value that S3 supports, including both
  key IDs and aliases.
```

**Review by jorgeorpinel — COMMENTED** [review]
Per https://github.com/iterative/dvc.org/pull/3533#pullrequestreview-972216611 (committing...)

**Comment by shcheklein** [comment]
@jorgeorpinel @dberenbaum ping

**Comment by jorgeorpinel** [comment]
I think this is ready.

**Review by dberenbaum — APPROVED** [review]

**Pull request merged**